plugins/generator-1.21.1/neoforge-1.21.1/templates/block/block.java.ftl

```diff
@@ -423,6 +423,18 @@ public class ${name}Block extends
 	}
 	</#if>
 
+	<#if hasProcedure(data.onEntityFallsOn)>

```

Comment:
Newly added triggers should ideally use new optimised dependencies system. See https://github.com/MCreator/MCreator/blob/00385b4c9bc604035985a938911678adce46c3f1/plugins/generator-1.21.1/neoforge-1.21.1/templates/block/block.java.ftl#L553 for example